packages/block-editor/README.md

```diff
@@ -348,6 +348,10 @@ _Returns_
 
 Undocumented declaration.
 
+### findInPresetsBy
+
+Undocumented declaration.
+
```

Comment:
I'll add docs for this once we've determined that this is, in fact, the best place for these utils 😅